README.md

```diff
@@ -19,4 +19,8 @@ Contributions welcome!
 - ตอบแทนบุญคุณแผ่นดิน
 - ควายแดง
 - สวะส้ม
+- ชั่ยๆ
```

Comment:
I think this word is too generic (I saw many twitter users tweet this), consider removing this line

Reply:
example : https://twitter.com/tanawatofficial/status/1317705543528587264